lib/advanced-open-file-view.coffee

```diff
@@ -20,6 +20,19 @@ isRoot = (inputPath) ->
   return path.dirname(inputPath) is inputPath
 
 
+absolutify = (inputPath) ->
+  ###
+  Ensure that the given path is absolute. Relative paths are treated as
+  relative to the current project root.
+  ###
+  if getRoot(inputPath) == "."
+    projectPaths = atom.project.getPaths()
+    if projectPaths.length > 0
+      return path.join(projectPaths[0], inputPath)
```

Comment:
Did we really want to just use the first path, or do we want to be a little cleverer, and try and see which project path matches the most of the input?  (i.e., if I've got a project with 'a/', and another with 'b/', and I try to create 'b/c', I'm betting I meant to create it in the second project root…)

We could also use the project root of the currently open file, if such exists…